pkg/tools/runopts.go

```diff
@@ -39,6 +39,7 @@ type RunOpts struct {
 	NoDocker        bool
 	Internal        bool
 	quiet           bool
+	StrictLoading   bool
```

Comment:
@satellite15 I added this to run opts instead of tool opts because `policy test` doesn't currently have tool opts in scope